libsolidity/formal/CVC4Interface.h

```diff
@@ -21,8 +21,16 @@
 
 #include <boost/noncopyable.hpp>
 
+#if defined(__GLIBC__)
+#define _GLIBCXX_PERMIT_BACKWARD_HASH
```

Comment:
This is ugly, but, unfortunately, I haven't found a way around it.

Reply:
What does it do? Can you add a comment?